### ISSUE TITLE
Upgrade from C99 -> C100 on `ARM x64` for `1.38.x - Nightly` (Checking C100)

### ISSUE THREAD

**Issue by kjozwiak** [brave/brave-browser#21741]
As per process, QA runs through the following cases to ensure that the major chromium bump that's about to get merged into `master` doesn't regress the `Nightly` channel. Once the major chromoium bump is merged into `master`, QA will run through a full manual pass on `Windows` & `Android`.

### Startup & Components

- [x] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler`, `Wireshark` or `LittleSnitch` (or a similar application)
  - [x] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
- [x] Restart the browser, load `brave://components`, wait for 8 mins and verify that no component shows any errors

**Note:** Always double check `brave://components` to make sure there's no errors/missing version numbers. We basically want to make sure that components are working as they regressed in C98 when we first merged into `master`.

### Upgrade

- [x] Make sure that data from the last version appears in the new version OK
- [x] Ensure that `brave://version` lists the expected Brave & Chromium versions
- [x] With data from the last version, verify that:
  - [x] Bookmarks on the bookmark toolbar and bookmark folders can be opened
  - [x] Cookies are preserved
  - [x] Installed extensions are retained and work correctly
  - [x] Opened tabs can be reloaded
  - [x] Stored passwords are preserved
  - [x] Sync chain created in previous version is retained
  - [x] Social media-blocking buttons changes are retained
  - [x] Rewards
    - [x] Wallet balance is retained
    - [x] Auto-contribute list is retained
    - [x] Both Tips and Monthly Contributions are retained
    - [x] Wallet panel transactions list is retained
    - [x] Changes to rewards settings are retained
    - [x] Ensure that Auto Contribute is not being enabled when upgrading to a new version if AC was disabled
  - [x] Ads
    - [x] Both `Estimated pending rewards` & `Ad notifications received this month` are retained
    - [x] Changes to ads settings are retained
    - [x] Ensure that ads are not being enabled when upgrading to a new version if they were disabled
    - [x] Ensure that ads are not disabled when upgrading to a new version if they were enabled

**Comment by Uni-verse** [comment]
Verification Completed using build:

```
1.38.52 Chromium: 100.0.4896.46 (Official Build) canary (32-bit) 
Revision	5ca33821b2211805855c77d334353d27c616a7ca-refs/branch-heads/4896@{#584}
OS	Android 12; Build/SP1A.210812.016
```

Device: Samsung GS 21

Version | Components
-|-
![Screen Shot 2022-03-17 at 2 53 49 PM](https://user-images.githubusercontent.com/17885425/158955673-3e319660-37e6-40d1-9c8a-88d177d9108b.png) | ![Screen Shot 2022-03-18 at 12 34 40 AM](https://user-images.githubusercontent.com/17885425/158956963-d92cbf23-3cad-4b1c-bb98-75a7d2c3afd5.png)

Triple checked and it seems like theres an error for one of the components. Not sure if this is expected.
